cisco_umbrella_dns/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
+    "manifest_version": "2.0.0",
+    "app_uuid": "9f98de10-9c98-4601-ae36-cbe25c4be018",
+    "app_id": "cisco-umbrella-dns",
+    "display_on_public_website": true,
+    "tile": {
+        "overview": "README.md#Overview",
+        "configuration": "README.md#Setup",
+        "support": "README.md#Support",
+        "changelog": "CHANGELOG.md",
+        "description": "Visualize Cisco Umbrella DNS Proxied and DNS Traffic.",
+        "title": "Cisco Umbrella DNS",
+        "media": [
+            {
+                "caption": "Cisco Umbrella DNS - DNS Traffic",
+                "image_url": "images/cisco_umbrella_dns_dns_traffic.png",
+                "media_type": "image"
+            },
+            {
+                "caption": "Cisco Umbrella DNS - Proxied Traffic",
+                "image_url": "images/cisco_umbrella_dns_proxied_traffic.png",
+                "media_type": "image"
+            }
+        ],
+        "classifier_tags": [
+            "Category::Log Collection",
+            "Category::Network",
+            "Submitted Data Type::Logs"
+        ]
+    },
+    "assets": {
+        "integration": {
+            "source_type_name": "cisco_umbrella_dns",
```

Comment:
For crawler-based integrations, we should add `"auto_install": false` to the `assets` > `integration` object.

Reply:
Done

Reply:
@bhargavnariyanicrest Next time you need to create a tile for a crawler based integration, you can use the `--type metrics_pull` with the `ddev create` command. The `auto_install` option will be automatically added to the manifest 🙂